public/static/js/filters/filters.common.js

```diff
@@ -471,6 +477,9 @@ filters.common.filter('conditional', function () {
       if (!data) {
         return null;
       }
+      if (!execution) {
```

Comment:
What scope is exection coming from?

At this point I don't see how it can ever not be undefined. Is this a cut and paste from line 504?

Reply:
Doh, the github UI concertina confused me!